polyphemus/lib/etls/ipi/ipi_etls.rb

```diff
@@ -17,7 +17,7 @@ def add_single_cell_linkers!
           process_watch_type_with(
             bucket('data')
               .watcher('single_cell_pool_processed')
-              .watch(/^single_cell_[^\/]*\/processed\/.*\/[^\/]+POOL[^\/]+\/.*$/),
+              .watch(/^single_cell_[^\/]*\/processed\/.*\/IPIPOOL[^\/]+(\/[^\/]*)?$/),
```

Comment:
Realized that all ipi record folders include IPI and that it improves accuracy of regexs.

Reply:
Connected with this comment below:
> These allow additional characters between "IPI"/"IPIPOOL" and the next "/" correct? I'm looking at folder structure and trying to double check what I can.

Reply:
Yes.  It requires only that the identifier start with IPI and IPIPOOl.  Tests cover this as well.